File: 08.WPFでSVGプレビューと印刷/CS/Common/Make単純なサンプル.cs
using Pao.Reports;

namespace SvgPreview
{
    static class Make単純なサンプル
    {
        public static void SetupData(IReport paoRep)
        {
            paoRep.LoadDefFile(Util.SharePath + "simple.prepd");

            paoRep.PageStart();
            paoRep.Write("Text2", "WPFデスクトップで作った\n印刷データですよ～ん♪");
            paoRep.PageEnd();
        }
    }
}

File: 08.WPFでSVGプレビューと印刷/VB/Common/Make単純なサンプル.vb
Imports Pao.Reports

Friend Module Make単純なサンプル

    Public Sub SetupData(paoRep As IReport)
        paoRep.LoadDefFile(Util.SharePath & "simple.prepd")

        paoRep.PageStart()
        paoRep.Write("Text2", "WPFデスクトップで作った" & vbLf & "印刷データですよ～ん♪")
        paoRep.PageEnd()
    End Sub

End Module
